scripts/capture-agent-network-sessions.py

```
@@ -0,0 +1,588 @@
+    if spec.mcp_config:
+        mcp_config = agent_dir / "mcp.json"
+        mcp_config.write_text('{"mcpServers":{}}')
+        command = [
+            *command[: command.index("--mcp-config") + 1],
+            str(mcp_config),
+            *command[command.index("--mcp-config") + 2 :],
+        ]
+
+    mitm_stdout = (agent_dir / "mitm.out").open("wb")
+    mitm_stderr = (agent_dir / "mitm.err").open("wb")
+    mitm = subprocess.Popen(
```

Ruff (0.15.13)

[error] `subprocess` call: check for execution of untrusted input

(S603)